GuildedRoseLLC/Details/DetailsViewController.swift

```diff
@@ -13,5 +15,16 @@ public class DetailViewController: UIViewController {
         }
         
         itemName.text = item?.name
+        sellIn.text = item?.sellIn ??? "None"
```

Comment:
OK. I think I see what is happening. It seems like you've defined your own custom operator of `???`. Is that accurate? What led to the introduction of this change, and what exactly does `???` do?